attributes/default.rb

```diff
@@ -37,6 +37,14 @@
   'crtfile' => 'self-signed.example.com.crt'
 }
 
+default['splunk']['clustering'] = {
+  'enable' => false,
+  'mode' => 'master', # master|slave|searchhead
```

Comment:
I agree; however, this is a common Splunk terminology:
http://docs.splunk.com/Documentation/Splunk/latest/Indexer/Enablethepeernodes
http://docs.splunk.com/Documentation/Splunk/latest/Admin/Serverconf
Either terminology works, but we may want to maintain consistency between Chef node attributes and Splunk config terms, in the interest of simplicity and avoiding confusion.